app/editor/src/features/admin/topics/Columns.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  FieldSize,
+  getEnumStringOptions,
+  ITableHookColumn,
+  ITopicModel,
+  OptionItem,
+  Select,
+  TopicTypeName,
+} from 'tno-core';
+
+export const Columns = (
```

Comment:
Coding standards for javascript are camel case for functions and variables.  Use capital letter for classes and components.  This is a function.